### PR TITLE
Backporting to version sans tropo corrections

### PR DESCRIPTION
**Pull request by sssangha**
But contains all other bug fixes/changes introducedin dev commit #4efd71c

### CODE REVIEW

**Comment by dbekaert** [comment]
@sssangha should be issuing PR to the v1 branch.

**Comment by sssangha** [comment]
@dbekaert whoops, my bad! fixed

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+version: 2.1
```
Comment (dbekaert) [inline .circleci/config.yml]: could you remove this file

**Comment by dbekaert** [comment]
@sssangha can you resolve commit issue?

**Review by dbekaert — APPROVED** [review]
@sssangha this looks good
Can you also update the extract jupyter notebook to point to the right DEM now.

**Pull request merged**